src/lib/utils/partners.ts

```diff
@@ -0,0 +1,44 @@
+import { shuffleArray } from "./array";
+
+const PARTNERS = [
+  "afpa",
+  "anlci",
+  "apprentisAuteuils",
+  "bge",
+  "bonsClics",
+  "bpi",
+  "cheops",
+  "collectifEmploi",
+  "cravateSolidaire",
+  "cresus",
+  "e2c",
+  "fondationFace",
+  "emploisInclusion",
+  "orange",
+  "socialBuilder",
+  "financesPedagogie",
+  "ligueEnseignement",
+  "positivePlanet",
+  "uniopss",
+  "forceFemmes",
+  "medNum",
+  "renault",
+  "wimoov",
+  "epide",
+  "google",
+  "mobin",
+  "serviceCivique",
+  "jeVeuxAider",
+  "monEnfant",
+  "simplon",
+  "konexio",
+  "nqt",
+  "snc",
+];
+
+export function getPartners(limit?: number) {
+  if (!limit) {
+    limit = PARTNERS.length;
+  }
+  return shuffleArray(PARTNERS).slice(0, limit);
```

Comment:
Si tu veux éviter une variable intermédiaire, autant faire 

```suggestion
  return shuffleArray(PARTNERS).slice(0, limit || PARTNERS.length);
```

De mon coté je ferais plutot
```
{
 const shuffledPartners =   return shuffleArray(PARTNERS);
 return limit ? shuffledPartners.slice(0, limit) : shuffledPartners;
}
```
Mais je peux vivre avec ta version si tu la préfères ;-)